ui/app/js/controllers/releases_controller.js

```diff
@@ -75,6 +77,11 @@ function($scope, $routeParams, $location, $timeout, Releases, Search, $modal, Pa
   }
   $scope.ordering_str = $scope.ordering_options[0];
 
+
+  $scope.$watch('pageSize', function(value) {
+    $scope.pageSize = value;
```

Comment:
Can you store and retrieve this in localStorage like we do for the product/channel drop down (https://github.com/mozilla/balrog/blob/d4b030dcf1c8fc097aaee6ffd7ac6366d70fcbf8/ui/app/js/controllers/rules_controller.js#L82). I suspect that different users may have different preferences for page size, so it would be nice to save them.